### ISSUE TITLE
Google forms design changes have breaked the code

### ISSUE THREAD

**Issue by aasaru** [devinrader/TwilioSheet#2]
Hi! It seems that Google has changed the design of its forms and the code doesn't work with new design. I found an existing form with old design and re-used that but it seems that at least the code on http://twiliosheet.azurewebsites.net/ isn't working with that either.

**Comment by rose-s** [comment]
Hey aasaru! Did you find a way to make the Twilio Sheet work with the new forms design? I'm trying to get it to work, too, with no luck so far.

**Comment by aasaru** [comment]
Nope. Since http://twiliosheet.azurewebsites.net/ wasn't working at all I gave up.

**Comment by binyomin** [comment]
I was able to get it to work by making the form an old style one. In the new form just select "Go back to the old Google Forms". And then use the url from the live old style form

**Comment by billywhite4** [comment]
Bummer, this is exactly what I'm looking for but running into the same problem. Selecting "Go back to the old Google Forms" no longer works either. It generates an identical URL that generates the error in TwilioSheet: "The URL you entered looks like a valid URL for a Google Form, I'm just having trouble validating it. Perhaps you entered the URL for an example form? "

**Comment by adenta** [comment]
What you have to do is go to:

File => Add Collaborators.

Then you copy that link.

**Comment by ghobbs003** [comment]
I tried the File => Add Collaborators link, but got the message 

"The URL you entered looks like a valid Google Form, but it doesn't have any inputs for Twilio data. Update your form to accept at least one Twilio parameter and try again." 

Pretty sure my form is right - it has SmsSid (Short Answer), From (Short Answer), Body (Paragraph). Tried them all as Paragraph as well. 

Tried setting access permissions to Anyone. Still no joy. 

Ideas?

**Comment by adenta** [comment]
In reply to my past comment: you must first follow the above steps to switch to the old Google forms. Once there, the add contributors buttons works!

**Comment by oJaison** [comment]
It's working! My steps: 
1. Convert to old Google Forms
2. Go to File => Add Collaborators
3. Edit collaborator link by replacing "/edit?usp=sharing" with "/viewform"
4. Use new link in [TwilioSheet](http://twiliosheet.azurewebsites.net/)
